include/fmt/format-inl.h

```diff
@@ -23,16 +23,15 @@
 #endif
 
 #ifdef _WIN32
-#  if defined(FMT_BIG_WIN) || defined(NOMINMAX)
-#    include <windows.h>
-#  else
-#    define NOMINMAX
-#    define WIN32_LEAN_AND_MEAN
-#    include <windows.h>
-#    undef WIN32_LEAN_AND_MEAN
-#    undef NOMINMAX
-#  endif
 #  include <io.h>
+FMT_BEGIN_NAMESPACE
+  using DWORD = uint32_t;
+  using HANDLE = void*;
```

Comment:
I suggest replacing the uses of `DWORD` and `HANDLE` with `uint32_t` and `void*` respectfully. Then `FMT_BEGIN/ED_NAMESPACE` can go away.

Reply:
In general I would avoid to define something in the global namespace. This could cause for example a conflict if you have windows.h included.
Removing DWORD and HANDLE is fine.